packages/app/templates/extensions/projects/flow_template/shopify.extension.toml.liquid

```diff
@@ -0,0 +1,31 @@
+name = "{{name}}"
+description = "A Flow Template example"
+type = "flow_template"
+
+[[extensions]]
+handle = "{{name}}"
+type = "flow_template"
+
+# Required -- Can be overridden by translations, but used in Partner dashboard
+title = "My Awesome Template"
+
+# Required -- Can be overridden by translations, but used in Partner dashboard
+description = "A Template that helps increase merchant productivity"
+
+# Required -- Must include one or more categories
+categories = ["orders", "fraud"]
+
+# Required
+pre_install_note = "You must disable automatic payment capture in the Shopify Admin before using this Template"
```

Comment:
Should these be localized?

Reply:
Yep, they are included in the localization paradigm. It's possible that we could remove them from here now and just rely entirely on translations. @cgidzinski what do you think?

Reply:
Its one solution though i feel it could be a better experience if we expose all the possible options instead of just hiding them in the localization file 🤔. That being said if our goal in the end is to translate many things then moving everything there would make sense.